build.gradle

```diff
@@ -99,9 +104,6 @@ allprojects {
 
 configurations {
     sqllite
-}
```

Comment:
This gave me a red squiggle in the gradle plugin for VS code since it was duplicated. If there was a reason for the dupe then I can revert this.